_posts/2023-11-20-curvature_and_remeshing.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New in CGAL: Interpolated Corrected Curvature Measures and Isotropic Remeshing with Sizing Field"
+description: ""
+category:
+tags: [""]
+---
+{% include JB/setup %}
+
+<p>
+The CGAL project has been participating to 12 editions of the <a hef="https://summerofcode.withgoogle.com/">Google Summer of Code</a>
+internship program. It is a great opportunity to welcome new members to our community, both contributors and mentors.
+Today, we highlight the inclusion into the library of a 2022 contribution from <a href="https://github.com/hoskillua">Hossam Mohamed Seed</a>
```

Comment:
Would it be okay to link to my LinkedIn instead of GitHub?
`<a href="https://www.linkedin.com/in/hossam-saeed-1abab91b3/">Hossam Mohamed Seed</a>`